src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4849,7 +4849,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
     <value>Tuple member name '{0}' is only allowed at position {1}.</value>
   </data>
   <data name="ERR_TupleReservedMemberNameAnyPosition" xml:space="preserve">
-    <value>Tuple membername '{0}' is disallowed at any position.</value>
+    <value>Tuple member name '{0}' is disallowed at any position.</value>
```

Comment:
Entered a bug https://github.com/dotnet/roslyn/issues/13419 to follow up. Should look through other errors/warnings.